contracts/src/v0.8/llo-feeds/VerifierProxy.sol

```diff
@@ -121,24 +121,30 @@ contract VerifierProxy is IVerifierProxy, ConfirmedOwner, TypeAndVersionInterfac
   }
 
   /// @inheritdoc IVerifierProxy
-  function verify(bytes calldata payload) external payable checkAccess returns (bytes memory) {
+  function verify(
+    bytes calldata payload,
+    bytes calldata feePayload
```

Comment:
Is this always going to be feePayload? or should we change this to parameterPayload - to save potential future confusion